lib/shared/config/loadfiles.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+var get = require('lodash.get');
+var set = require('lodash.set');
+var isString = require('lodash.isstring');
+var configSpecs = require('./cfg-specs');
+
+function loadConfigFiles(configFilesBase, keysInOrder) {
+  var config = {};
+
+  var configFilePaths = keysInOrder.map(function(key) {
```

Comment:
Properties order in objects is not guaranteed, though `Object.keys` looks like returning in definition order actually. I think it is needed to specify the order explicitly.